pandas/tests/test_util.py

```diff
@@ -150,6 +151,38 @@ def test_validation(self):
         kwargs = {'f': 'foo', 'b': 'bar'}
         validate_kwargs('func', kwargs, *compat_args)
 
+
+class TestMove(tm.TestCase):
+    def test_more_than_one_ref(self):
+        """Test case for when we try to use ``move_into_mutable_buffer`` when
+        the object being moved has other references.
```

Comment:
its funny, when you use a doc-string it calls the test by this. I guess that is right.